pkgs/development/python-modules/trezor/default.nix

```diff
@@ -1,27 +1,51 @@
-{ lib, fetchPypi, buildPythonPackage,
-  protobuf, hidapi, ecdsa, mnemonic, requests, pyblake2, click, libusb1, rlp, isPy3k
+{ lib, fetchPypi, buildPythonPackage, isPy3k, python, pytest, glibcLocales
+, typing-extensions
+, protobuf
+, hidapi
+, ecdsa
+, mnemonic
+, requests
+, pyblake2
+, click
+, construct
+, libusb1
+, rlp
 }:
 
 buildPythonPackage rec {
   pname = "trezor";
-  version = "0.10.2";
+  version = "0.11.1";
 
   disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "4dba4d5c53d3ca22884d79fb4aa68905fb8353a5da5f96c734645d8cf537138d";
+    sha256 = "6043f321d856e1b45b9df0c37810264f08d065bb56cd999f61a05fe2906e9e18";
   };
 
-  propagatedBuildInputs = [ protobuf hidapi ecdsa mnemonic requests pyblake2 click libusb1 rlp ];
+  nativeBuildInputs = [ glibcLocales ];
 
-  # There are no actual tests: "ImportError: No module named tests"
-  doCheck = false;
+  propagatedBuildInputs = [ typing-extensions protobuf hidapi ecdsa mnemonic requests pyblake2 click construct libusb1 rlp ];
+
+  # build requires UTF-8 locale
+  LANG = "en_US.UTF-8";
+  LOCALE_ARCHIVE = "${glibcLocales}/lib/locale/locale-archive";
```

Comment:
Is `LOCaLE_ARCHIVE` still required?

Reply:
no, fixed.

Reply:
Any reason for this to be backported?

Reply:
Couple of them:

1) we add a lot of features into Trezor and these are not accessible if a user has an older version of python-trezor leading to frustration
2) I assume NixPkgs Electrum maintainers don't backport security patches individually, but rather the whole package is being updated to the newest stable release. This release might already be depending on the new python-trezor package, using the new API, etc. so it makes sense to upgrade the whole package for python-trezor too.